presto-tests/src/main/java/io/prestosql/tests/AbstractTestDistributedQueries.java

```diff
@@ -847,6 +848,26 @@ public void testShowCreateView()
         assertUpdate("DROP VIEW " + viewName);
     }
 
+    @Override
+    public void testAccessControl()
```

Comment:
let's have a separate test for this `testViewAccessControl`

Reply:
there already was one, added these tests to it